### ISSUE TITLE
MooTools incompatibility

### ISSUE THREAD

**Issue by fritzmg** [ErdmannFreunde/euf_overlay#12]
All the JavaScript files are missing a `$`, causing problems when MooTools is enabled as well.
```diff
-jQuery(document).ready(function() {
+jQuery(document).ready(function($) {
```

**Comment by tarun-uwe** [comment]
Sehr nice. Schneller Fix. Danke @fritzmg

**Comment by denniserdmann** [comment]
fixed in b20101276b93444b1aea59d55baf6c3ade2fc3f8
